station/submitobs.py

```diff
@@ -59,9 +60,10 @@ def get_tle(sat_name: str, date: datetime.datetime) -> Optional[List[str]]:
     try:
         db = OrbitDatabase()
         return db.get_tle(sat_name, date)
-    except:
+    except BaseException:
```

Comment:
We should use `Exception` instead of the `BaseException` here and in similar places.

`BaseException` catches all `Exception` subclasses and additionally `SystemExit`, `KeyboardInterrupt`, and `GeneratorExit`. Usually, you don't want to catch the `*Exit` or `KeyboardInterrupt` exceptions because it blocks the possibility of terminating the application.

Reply:
You're right. I've updated it as suggested.

Reply:
Also updated all other instances.